js/views/reader_view.js

```diff
@@ -547,11 +547,7 @@ ReadiumSDK.Views.ReaderView = function(options) {
 
                 var spineItem = _spine.getItemById(bookMark.idref);
                 
-                initViewForItem(spineItem, function(isViewChanged){
-
-                    if(!isViewChanged) {
-                        _currentView.setViewSettings(_viewerSettings);
-                    }
+                initViewForItem(spineItem, function(isViewChanged) {
```

Comment:
@danielweck 
Is this issue fixed by: https://github.com/readium/readium-shared-js/pull/192
?